src/controller/SetUpCodePairer.cpp

```diff
@@ -104,26 +104,15 @@ CHIP_ERROR SetUpCodePairer::Connect()
         if (ShouldDiscoverUsing(RendezvousInformationFlag::kBLE))
         {
             CHIP_ERROR err = StartDiscoveryOverBLE();
-            if (err != CHIP_NO_ERROR)
+            if ((err != CHIP_NO_ERROR) && (err != CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE))
```

Comment:
Why this change?  Knowing that BLE is unsupported in the build and that's why BLE discovery did not happen is in fact useful.